src/v2/guide/instance.md

```diff
@@ -31,42 +31,44 @@ Root Instance
       └─ TodoListStatistics
 ```
 
-We'll talk about [the component system](components.html) in detail later. For now, just know that all Vue components are also Vue instances, and so accept the same options object (except for a few root-specific options).
+ما در مورد [سیستم componentها](components.html) بعدا با جزئیات صحبت می‌کنیم. اما در حال حاضر فقط بدانید که هر component ویو در واقع یک vue instance هم هستند و بنابراین آپشن‌های مشابهی قبول می‌کنند. (به جز تعداد کمی از آپشن‌های مخصوص root.)
 
-## Data and Methods
+## Data و Methodها
 
-When a Vue instance is created, it adds all the properties found in its `data` object to Vue's **reactivity system**. When the values of those properties change, the view will "react", updating to match the new values.
+وقتی یک instance ویو ساخته می‌شود، ویو تمام propertyهای شئ `data` را به **سیستم واکنشگرایی (reactivity)** خود اضافه می‌کند. وقتی مقدار این propertyها تغییر می‌کند، صفحه «واکنش» نشان داده و متناسب با مقدار جدید بروز می‌شود.
 
 ```js
-// Our data object
+// شئ data ما
 var data = { a: 1 }
 
-// The object is added to a Vue instance
+// شئ به یک instance از ویو اضافه شده‌است.
 var vm = new Vue({
   data: data
 })
 
-// Getting the property on the instance
-// returns the one from the original data
+
+// با گرفتن یک property از instance
```

Comment:
Comments will be rendered wrongly:

![image](https://user-images.githubusercontent.com/5158436/60462391-62cd6e80-9c5e-11e9-8bf0-a33fa110aa66.png)

Reply:
I've noticed this problem and I was wondering to add a style fix for all comments to change their direction. But I'm not sure if it is a good idea.
Otherwise, we can leave comments to stay English since in real codes comments are often in English.

Reply:
Here comments are part of the tutorial, so I think it's better to translate them as well.
changing style is an extra work but probably is the best thing to do.
In my translation I typed in the wrong order to make it look right on output, but it's not a good approach for accessibility since screen readers will also read it in the wrong order.